php/worker.md

```diff
@@ -119,4 +126,4 @@ first input character.
 ## Other Type of Workers
 
 Different roadrunner implementations might define their own worker APIs,
-examples: [GRPC](https://github.com/spiral/php-grpc), [Workflow/Activity Worker](https://docs.temporal.io/docs/php-workers/).
\ No newline at end of file
+examples: [GRPC](https://github.com/spiral/php-grpc), [Workflow/Activity Worker](https://docs.temporal.io/docs/php-workers/).
```

Comment:
Hey @jjsaunier . BTW, could you please include new GRPC repo instead of old `php-grpc` ? -> https://github.com/spiral/roadrunner-grpc